courses/machine_learning/deepdive2/structured/solutions/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -0,0 +1,814 @@
+{
```

Comment:
<p>see above re Learning Objective #5 and opening sentence.</p><p> </p><p>Remove last sentence. This is the solution notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>In all of the other solutions notebooks I kept this line in.</p>